components/support/ktx/src/test/java/mozilla/components/support/ktx/kotlin/StringTest.kt

```diff
@@ -84,30 +83,6 @@ class StringTest {
         assertTrue("Geo:1,-1".isGeoLocation())
     }
 
-    @Test
-    fun toUri() {
```

Comment:
Nit: As long as the extension method exists (even though `@Deprecated`), we should keep the test too.